release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -622,6 +622,13 @@ In {product-title} {product-version}, CoreDNS uses version 1.10.0, which include
 * CoreDNS now always prefixes each log line in Kubernetes client logs with the associated log level.
 * CoreDNS now reloads more quickly at an approximate speed of 20ms.
 
+[id="ocp-4-12-ne-reload-interval"]
+==== Support for a configurable reload interval in HAProxy
+
+With this update, a cluster administrator can configure the reload interval to force HAProxy to reload its configuration less frequently in response to route and endpoint updates. The default minimum HAProxy reload interval is 5 seconds.
```

Comment:
Not critical, but "With this update" sounds like something that would be in a z-stream instead of a minor release note. I would expect something either like "with this release", or that that "Cluster administrators can now" or "OCP now supports".

But I see a few other places that use it in the enhancements section, so not a big deal. Doesn't look like something we really standardize on, just figured I'd mention it.